packages/quick_actions/quick_actions/android/src/test/java/io/flutter/plugins/quickactions/QuickActionsTest.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.quickactions;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import io.flutter.embedding.engine.plugins.FlutterPlugin.FlutterPluginBinding;
+import io.flutter.plugin.common.BinaryMessenger;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.StandardMethodCodec;
+import java.nio.ByteBuffer;
+import org.junit.Test;
+
+public class QuickActionsTest {
+  private static class TestBinaryMessenger implements BinaryMessenger {
+    public MethodCall lastMethodCall;
```

Comment:
Do you want to add a test demonstrating using this?

Reply:
I wanted to, but the plugin doesn't actually call `MethodChannel.invokeMethod` anywhere yet. This was mostly added for https://github.com/flutter/plugins/pull/4204.